lineapy/transformer/node_transformer.py

```diff
@@ -188,7 +188,10 @@ def visit_NameConstant(self, node: ast.NameConstant) -> LiteralNode:
             return self.tracer.literal(node.value, self.get_source(node))
 
     def visit_Raise(self, node: ast.Raise) -> None:
-        return super().visit_Raise(node)
+        try:
+            return super().visit_Raise(node)
```

Comment:
[visit_Raise missing in Python 3.7](https://github.com/python/cpython/blob/3.7/Lib/ast.py)
[visit_Raise missing in Python 3.8](https://github.com/python/cpython/blob/3.8/Lib/ast.py)

but it exists in [3.9+](https://github.com/python/cpython/blob/3.9/Lib/ast.py#L935)

 Perhaps using `sys.version_info` is better than a try catch here.

Reply:
The method tagged for 3.9 isn't the right one. its part of an internal example class used for testing called unparser (and not the base ast.NodeTransformer class). figuring out how super().visit_Raise(node) gets resolved will help you get to the bottom of this.

Reply:
Ok I see, if this is the case then this should never work, and we should just do `self._exec_statement(node)` in all cases right? @LordDarkula can we confirm which versions of python this actually affects and go from there?

Reply:
It seems like this fails in all versions of Python.